### ISSUE TITLE
Fix settings to pass testDefaultImageDoesNotStoreAnySetting ?

### ISSUE THREAD

**Issue by tinchodias** [npasserini/iceberg#1405]
The [PR to merge into Pharo 9.0](https://github.com/pharo-project/pharo/pull/8461) showed some tests failures.
One of them is `testDefaultImageDoesNotStoreAnySetting`, which I don't understand well, but finds that the following settings have something wrong:

```
a StoredSetting('#useCustomSsh#username')
a StoredSetting('#useCustomSsh#keyPassphrase')
a StoredSetting('#commitPreferences#saveImageOnCommit')
a StoredSetting('#commitPreferences#pushOnCommit'))
```

This seems to be showstopper to proceed to merge v2 so must be fixed with priority.

**Comment by tinchodias** [comment]
I just checked that the first 2 settings already make this test fail in P9.0. 

By contrary, the last two are added by v2.

**Comment by tesonep** [comment]
@tinchodias does the settings have default values? Are the default values the same set "by default" by the initialization.
The idea of the test is to guarantee that a clean image does not save settings when you save settings (there all defaults, and only the non-defaults are saved).

**Comment by tesonep** [comment]
@tinchodias ping me and we check it together if you wish

**Comment by tinchodias** [comment]
Maybe these settings should be stored in the credentials store. I can ping you later.

**Comment by tinchodias** [comment]
We connected and fixed it, in dev-2.0

https://github.com/pharo-vcs/iceberg/commit/e73466490fd145b8eb072deca40861d148f2cca1
